Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: quality style test docs utils
 
-check_dirs := tests src examples benchmarks utils
```

Comment:
This should remain unchanged

Reply:
Why? Not having this here makes it easier to run Ruff on the command line (for instance to enable more lint rules) without having to remember which directories to include or exclude.

Reply:
Makes sense!